samples/subsys/usb/midi/README.rst

```diff
@@ -0,0 +1,81 @@
+.. zephyr:code-sample:: midi
+   :name: USB MIDI device class sample
+   :relevant-api: usbd_api usb_midi
+
+   USB MIDI 2.0 device class sample
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/templates/sample.tmpl#L5-L6

```suggestion
   Implement a simple USB MIDI 2.0 device that loops back MIDI messages and sends note events using a user button.
```

bonus points if you find a way to make it shorter. "Implement a simple USB MIDI 2.0 device" would probably work, actually.